tests/framework/console/controllers/MigrateControllerTest.php

```diff
@@ -50,4 +50,34 @@ protected function getMigrationHistory()
         $query = new Query();
         return $query->from('migration')->all();
     }
+
+    public function testSessionMigration()
```

Comment:
That is hard to tell.
We do not have any unit tests for the migrations provided by the core.
I am unsure if they are necessary or not.
But if adding it - it should be a separated test case.